influent-java/src/main/java/influent/forward/NioSslForwardConnection.java

```diff
@@ -0,0 +1,129 @@
+package influent.forward;
+
+import influent.internal.msgpack.MsgpackStreamUnpacker;
+import influent.internal.nio.NioAttachment;
+import influent.internal.nio.NioEventLoop;
+import influent.internal.nio.NioSslChannel;
+import influent.internal.util.ThreadSafeQueue;
+import org.msgpack.core.MessageBufferPacker;
+import org.msgpack.core.MessagePack;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.SSLContext;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.SelectionKey;
+import java.nio.channels.SocketChannel;
+
+public class NioSslForwardConnection implements NioAttachment {
```

Comment:
Can this class is removed if `NioSslChannel` handles all the encryption and decryption?